docs/faq/wallets-and-seeds.md

```diff
@@ -70,27 +70,7 @@ A public key address, also called Pay-To-Pubkey (P2Pk), can be identified with i
 
 #### 8. What are the different types of addresses?
 
-A Decred address[^14995] is actually just a representation of a public key (which itself could be a script hash) along with a 2-byte prefix which identifies the network and type and a checksum suffix in order to detect improperly entered addresses.
```

Comment:
As we are dropping this line, we no longer need the reference [^14995] at the bottom of this file. We can delete that line as well